src/krfctl/krfctl.c

```diff
@@ -131,7 +131,9 @@ int main(int argc, char *argv[]) {
              " -T <variable>=<value>       enable targeting option <variable> with value <value>\n"
              " -C                          clear the targeting options\n"
              "targeting options:\n"
-             " personality, PID, UID, GID, and INODE\n");
+             " personality, PID, UID, GID, and INODE\n"
+             "available profiles (for -P flag):\n"
+             " fs, io, ipc, memory, mm, net, proc, sched, sys, time\n");
```

Comment:
Let's loop over the `fault_profile_table` instead of hardcoding these, so that we don't have to remember to update them :slightly_smiling_face:

Reply:
Done :)